tests/README.md

```diff
@@ -0,0 +1,90 @@
+MetalK8s is an infrastructure product, that rely on ansible so framework in python is somewhat “obvious”.
+Pytest is quite famous right famous.
+
+Test to be run could be splitted in different category:
+
+1. Test that run before installation on any machine
+    - test of os-check (centos check and kernel)
+    - test first installation of component that doesn't need kubernetes (node_exporter)
+    - test of the complete installation it-self
+
+2. Test that run after the installation of kubernetes only (not the entire set of MetalK8s):
```

Comment:
I don't understand this.

Reply:
For exemple role kube_prometheus install prometheus via helm, when you don't want to test an upgrade, you need test in "isolation" like you already mentioned. But for kube_prometheus we require kubernetes to be up. May be we could say that we can't test the  `.2`

Reply:
Yeah, roles that interact with K8s are... tricky :-/ Doesn't fit the Ansible model all too well anyway.